typescript/phoenix-sdk/src/market.ts

```diff
@@ -234,4 +226,30 @@ export class Market {
       inAmount,
     });
   }
+
+  /**
+   * Subscribes to updates on the market
+   *
+   * @param connection The Solana `Connection` object
+   */
+  async subscribe(connection: Connection) {
+    const subId = connection.onAccountChange(this.address, (account) => {
+      const marketData = deserializeMarketData(account.data);
+      this.data = marketData;
+    });
+    this.subscriptions.push(subId);
+  }
+
+  /**
+   * Unsubscribes from updates when the market is no longer needed
+   *
+   * @param connection The Solana `Connection` object
+   */
+  unsubscribe(connection: Connection) {
+    for (const subId of this.subscriptions) {
+      connection.removeAccountChangeListener(subId);
```

Comment:
does this need to be `await`ed?

https://github.com/solana-labs/solana-web3.js/blob/a0ec66e/src/connection.ts#L6230